qbft/spectest/generate/main.go

```diff
@@ -37,6 +41,59 @@ func main() {
 
 	fmt.Printf("found %d tests\n", len(all))
 	writeJson(byts)
+
+	for _, testF := range spectest.AllTests {
+		test := testF()
+		// generate post state comparison
+		post, err := test.GetPostState()
+		if err != nil {
+			panic(err.Error())
+		}
+		writeJsonStateComparison(test.TestName(), reflect.TypeOf(test).String(), post)
+	}
+}
+
+func clearStateComparisonFolder() {
+	_, basedir, _, ok := runtime.Caller(0)
+	if !ok {
+		panic("no caller info")
+	}
+	dir := filepath.Join(strings.TrimSuffix(basedir, "main.go"), "state_comparison")
```

Comment:
`main.go` seems redundant here, `filepath.Join(basedir, "state_comparison")` would give the same result